ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -87,6 +89,8 @@ struct MutexAndUseCount
 
 typedef map<FsidInode, MutexAndUseCount, FsidInodeCompare> FileMutexMap;
 
+static fsid_t virtualizationRoot_fsid;
```

Comment:
I believe our coding guidelines require this to start with `s_` and be camelCase past that.